### PR TITLE
Kp nucleus cyber rename

### PR DESCRIPTION
**Pull request by K-Patel-NC**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Review by NikTripathi — CHANGES_REQUESTED** [review]

**Comment by NikTripathi** [comment]
This is an update to an existing solution. So any changes made should be a version increment.

**Comment by K-Patel-NC** [comment]
> ny changes made should be a version increment.

File: Solutions/archTIS/Data Connectors/Logo/NCProtectLogo.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (K-Patel-NC) [reply Solutions/archTIS/Data Connectors/Logo/NCProtectLogo.svg:1]: Hi Nik, 

Sorry by mistake, I closed pull request. 

Few question regarding,

I can see Test case pass for logoValidator and do we really require to logo size should be 5kb.
Also We haven't release our solutions, do we really require to increase version?

[the remark this answers is not in the document]
Comment (K-Patel-NC) [reply Solutions/archTIS/Data Connectors/Logo/NCProtectLogo.svg:1]: Is there any way you can reopen pull request?

[the remark this answers is not in the document]
Comment (K-Patel-NC) [reply Solutions/archTIS/Data Connectors/Logo/NCProtectLogo.svg:1]: Reopen again.

File: Solutions/archTIS/Data Connectors/Logo/NCProtectLogo.svg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.0//EN" "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="body_1" width="200" height="19">
```
Comment (NikTripathi) [inline Solutions/archTIS/Data Connectors/Logo/NCProtectLogo.svg:3]: Id should be guid.

**Review by NikTripathi — CHANGES_REQUESTED** [review]

**Comment by NikTripathi** [comment]
Please check the PR validations.

File: Solutions/archTIS/Data Connectors/Logo/NCProtectLogo.svg (hunk 2 above)
Comment (K-Patel-NC) [reply Solutions/archTIS/Data Connectors/Logo/NCProtectLogo.svg:3]: Hi @NikTripathi , Just wondering is data:image should be fine?

Comment (NikTripathi) [reply Solutions/archTIS/Data Connectors/Logo/NCProtectLogo.svg:3]: According to parameters there should not be xlinks.

Comment (K-Patel-NC) [reply Solutions/archTIS/Data Connectors/Logo/NCProtectLogo.svg:3]: Hi @NikTripathi , Thanks for your update. I am waiting for proper svg image without data:image as well as xlinks and it should be less than 5kb from business.

**Comment by K-Patel-NC** [comment]
@NikTripathi , Looks like KqlValidations is failing which was passed before and I didn't have change anything except logo.
Looks like something has been changed on kqlValidationTest. Can you please tell me do I need to investigate on my side or something is broken on your side.

**Comment by NikTripathi** [comment]
> @NikTripathi , Looks like KqlValidations is failing which was passed before and I didn't have change anything except logo.
> Looks like something has been changed on kqlValidationTest. Can you please tell me do I need to investigate on my side or something is broken on your side.

@K-Patel-NC KqlValidations is failing from our side. We are working to get it fixed. No need to investigate on your side.

**Review by v-rucdu — APPROVED** [review]

**Pull request merged**